### ISSUE TITLE
content: seed the platform layer between a trained model and a served one

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#300]
The corpus explains how a model is trained (`pretraining`, `fine-tuning`, `data-parallelism`) and how it is served (`continuous-batching`, `paged-attention`, `prefill-and-decode`), and has almost nothing about the machinery in between:

```
feature-store           1 mention, no node
training-serving skew   0 mentions
experiment-tracking     0            data-versioning        0
online-learning         0            batch-inference        0
offload                 0            disaggregated-serving  0
```

**`training-serving skew` is the sharpest absence.** It is the failure mode that makes feature stores exist — a feature computed one way in training and another way at serving — and it appears nowhere, despite `distribution-shift`, `silent-failure` and `production-monitoring` all being downstream of it.

`offload` is the other one that costs something: `zero-redundancy-optimizer` and `activation-checkpointing` both trade memory for something else, and the technique that trades it for host memory or NVMe is not here.

The four "disaggregat" hits in the corpus are all about fairness disaggregation, so `disaggregated-serving` — splitting prefill and decode onto separate hardware — is genuinely absent rather than merely unnamed.

## Scope — eight nodes

`feature-store` · `training-serving-skew` · `experiment-tracking` · `data-versioning` · `online-learning` · `offload` · `disaggregated-serving` · `batch-inference`

Scoping constraints:

- `online-learning` must be scoped against `continual-learning`, which is about forgetting across tasks. This is updating from a live stream, where the constraint is that the data arrives once and in order.
- `batch-inference` must be scoped against `continuous-batching`, which is a serving-time scheduling technique. This is the offline mode — throughput over latency, and the different system that implies.
- `data-versioning` must not restate `data-provenance`. That is where the data came from; this is being able to name and retrieve the exact snapshot a run used.

## Acceptance criteria

- [ ] Eight nodes, schema-valid, connected — no restatement of `continual-learning`, `continuous-batching`, `data-provenance` or `model-versioning`
- [ ] Every arXiv id verified by title before being written
- [ ] Figures assigned from arithmetic; captions name their control
- [ ] `make golden` regenerated; all gates green
- [ ] Verified in a second pass that re-derives each math body
- [ ] Teal lobe moved when the verification empties it

Refs #61, #288
